polistackapp/templates/bills.html

```diff
@@ -12,29 +12,41 @@
             margin: 0;
             padding: 20px;
         }
-
         .logo {
             font-size: 30px;
         color: orangered;
         }
-
         .btn-primary {
             background: orangered;
             border: 1px solid #e73b00;
         }
-
+        .btn-primary:hover {
+         background: #ff7440;
+         border: 1px solid #e73b00;
+         }
         button:disabled {
```

Comment:
Same for this, it should be moved to a separate CSS file.

Reply:
yup makes sense.